app/src/standard/AndroidManifest.xml

```diff
@@ -35,6 +35,21 @@
                 <!-- see Samsung multi-window info at bottom -->
                 <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER" />
             </intent-filter>
+            <intent-filter>
```

Comment:
andbible.org could be auto-enabled if using `android:autoVerify="true"` but it would query automatically https://andbible.org/.well-known/assetlinks.json which we do not want.

Reply:
andbible.github.io is not an option for autoverify, because github.io does not serve files starting with dot.